templates/lightning/list_transactions.html

```diff
@@ -20,6 +20,9 @@ <h2>BTC TRANSACTIONS</h2>
 		<td>
 		{% for output in tx.outputs %}
 			<a href="{{ bitcoin_explorer }}tx/{{ tx.hash }}">{{ output.sats }},<a>
+			{% if loop.last %}
```

Comment:
why repeat the last output?

Reply:
this is to remove the "," at the list of output.

Reply:
rearranging it to shorten.